### PR TITLE
Create category pages

### LINKED ISSUE #165: Create /category pages
- [ ] Use the same presentation as `archives#index`
- [ ] Extract Article listing from `archives#index` into a partial
- [ ] Route: `/categories/:slug`

### PR DESCRIPTION
**Pull request by guitsaru**
- [x] Use the same presentation as archives#index
- [x] Extract Article listing from archives#index into a partial
- [x] Route: /categories/:slug

Closes #165

### CODE REVIEW

**Comment by veganstraightedge** [comment]
Looks great. I've got just a couple few nit picky things that I'll leave as a review on the code (mostly whitespace), then it's good to merge. Thanks!

File: app/views/archives/index.html.erb [hunk 1]
```diff
@@ -18,32 +18,7 @@
...
-                  <footer>
-                    <%= render "articles/tags",       article: article %>
-                  </footer>
-                </div>
-              </div>
-            </article>
-          </li>
-        <% end %>
-      </ol>
+      <%= render "shared/article_list", articles: articles %>
```
Comment (veganstraightedge) [inline app/views/archives/index.html.erb]: Will you move this to `articles/_list` instead, please? 🦊

File: app/views/categories/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<header>
+  <h1>Category</h1>
+
```
Comment (veganstraightedge) [inline app/views/categories/show.html.erb]: This line break can get deleted.

File: app/views/categories/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  <h2><%= @category.name %></h2>
+</header>
+
+<div class="h-feed">
+
```
Comment (veganstraightedge) [inline app/views/categories/show.html.erb]: This line break can get deleted.

File: app/views/categories/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  <%= render "shared/article_list", articles: @articles %>
+
```
Comment (veganstraightedge) [inline app/views/categories/show.html.erb]: This line break can get deleted.

**Comment by guitsaru** [comment]
Should be good to go now

**Comment by veganstraightedge** [comment]
:shipit:

**Pull request merged**